src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -640,10 +640,13 @@ function tryRemoveElement(elem) {
 
             // https://msdn.microsoft.com/en-us/library/hh772507(v=vs.85).aspx
 
+            /**
+             * @type {HTMLOptionsCollection|any[]}
```

Comment:
```suggestion
             * @type {AudioTrackList|any[]}
```

Reply:
That type doesn't exist

Reply:
I fixed it at `ArrayLike<any>` for now

Reply:
At least it doesn't affect the execution.
Some info https://github.com/microsoft/TypeScript/issues/38544

Reply:
Yeah it probably needs some time to tickle down, now every IDE probably already has v3.9.3 yet for now I leave it like that once it's ts it'll be correctly inferred anyway

Reply:
Oh if I read it correctly it hasn't been in closed at all yet, that makes it more difficult anyway but that's something for when ts is there